src/re_posh/subs.cljc

```diff
@@ -183,3 +187,14 @@
      {:type    :pull
       :pattern pattern
       :id      id})))
+
+(defn reg-pull-many-sub
+  "Syntax sugar for writing pull-many queries.
+  Same as reg-pull-sub but takes vector of eids under key :ids"
```

Comment:
Could you please add a code example here?

Reply:
Added a code example. Though I don't expect using `reg-pull-many-sub` will be especially common, as you probably want to execute a query & then pull those IDs in the subscription layer... a combination function of query + pull-many may be a useful addition at some point.